.gitignore

```
@@ -25,3 +25,5 @@ yarn-error.log*
 __generated__
 .vscode
 .idea
+
+.env
```

‫فقط در پوشه داکر .env را ایگنور کنید، در بقیه 

```suggestion
/docker/.env
```